admin/includes/modules/copy_product_confirm.php

```diff
@@ -70,6 +70,12 @@
           'product_is_call' =>  'int', 
           'products_quantity_mixed' =>  'int', 
         ); 
+        
+        // -----
+        // Give an observer the chance to add any customized fields to the two arrays above.
+        //
+        $zco_notifier->notify('NOTIFY_MODULES_COPY_PRODUCT_CONFIRM_DUPLICATE_FIELDS', $products_id, $separately_updated_fields, $casted_fields);
```

Comment:
Would it make sense to use `$product` instead of `$products_id` here, since there's probably a desire to inspect multiple fields, such as product-type?